### ISSUE TITLE
Unable to connect to localhost minutes after opening wrapper

### ISSUE THREAD

**Issue by jm1996117** [Wrapper-Offline/wrapper-offline#120]
For some reason, wrapper offline will be unable to save my video or generate TTS clips after a few minutes of the program being open. TTS will be stuck on "generating" and when saving a video I will get an error saying something like there was a problem. When i go to reopen the video list, I get an error message saying that the localhost refused to connect.

**Comment by GoTest334** [comment]
Are you sure you are using the latest stable version of wrapper (1.2.3)? I recommend you re download it if you're not. That should fix it.

**Comment by jm1996117** [comment]
i have confirmed I am using version 1.2.3. typically the issue will occur a few minutes after at least 1 TTS message has been generated.

**Comment by jm1996117** [comment]
I ran wrapper with verbose mode on and this is what I got:
> wrapper-offline@1.2.3 start C:\Users\HP\Downloads\wrapper-offline-1.2.3\wrapper-offline-1.2.3\wrapper
> node main.js


(node:8788) UnhandledPromiseRejectionWarning: Error: Could not connect
    at Socket.onerror (C:\Users\HP\Downloads\wrapper-offline-1.2.3\wrapper-offline-1.2.3\wrapper\node_modules\discord-rpc\src\transports\ipc.js:32:16)
    at Object.onceWrapper (events.js:422:26)
    at Socket.emit (events.js:315:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
(node:8788) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8788) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:8788) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
C:\Users\HP\Downloads\wrapper-offline-1.2.3\wrapper-offline-1.2.3\wrapper\node_modules\brotli\build\encode.js:3
1<process.argv.length?process.argv[1].replace(/\\/g,"/"):"unknown-program");b.arguments=process.argv.slice(2);"undefined"!==typeof module&&(module.exports=b);process.on("uncaughtException",function(a){if(!(a instanceof y))throw a;});b.inspect=function(){return"[Emscripten Module object]"}}else if(x)b.print||(b.print=print),"undefined"!=typeof printErr&&(b.printErr=printErr),b.read="undefined"!=typeof read?read:function(){throw"no read() available (jsc?)";},b.readBinary=function(a){if("function"===
                                                                                                                                                                                                                              ^

SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at IncomingMessage.<anonymous> (C:\Users\HP\Downloads\wrapper-offline-1.2.3\wrapper-offline-1.2.3\wrapper\tts\load.js:237:45)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (_stream_readable.js:1221:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 7
npm ERR! wrapper-offline@1.2.3 start: `node main.js`
npm ERR! Exit status 7
npm ERR!
npm ERR! Failed at the wrapper-offline@1.2.3 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\HP\AppData\Roaming\npm-cache\_logs\2021-07-01T00_07_18_275Z-debug.log
